### DISCUSSION TITLE
Is Origin theme Subscription ready?

### DISCUSSION THREAD

**Discussion by aanomm** [swellstores/community#320] [Help] [answered]
Hello

- I noticed the Swell Origin theme demo doesn't have any [frontend] subscription offerings; not even for [the products in the Subscription coffee category](https://origin-demo.swell.store/categories/subscriptions/)
- Is Origin theme Subscription ready?

**Comment by steven-k1** [comment, marked as answer]
Hi @fucha-co, The origin theme does not offer the same out-of-the-box subscription experience similar to the Horizon theme, but it does support products with subscription options. If a product is set up with the option for a subscription, it will show an additional option to choose when adding it to the cart. (screenshot below)

![image](https://github.com/swellstores/community/assets/77881204/4a6af122-adf1-4a9d-af59-5fbaba6fbe47)

**Reply by holden** [reply]
Does that mean that for the option / variant pricing to effect the subscription price (as it does single purchases), this won't work on Origin as it does in Horizon? eg https://swell-horizon-demo.vercel.app/products/moisturizer-spf-15

Is this feature specific to Horizon and there's some additional logic that one would need to add in the theme to get the same behavior in Origin?

**Reply by ericingram** [reply]
All of Swell's pricing logic is implemented on the backend API, so any functional difference is due to the implementation of Origin/Horizon/etc

**Reply by aanomm** [reply]
can i ask why Origin demo doesn't showcase any subscription features then? it's seems logical that it would show subscriptions if it's capable subscriptions; espeically since subscriptions are such a huge feature of Swell?

**Reply by holden** [reply]
In the Horizon theme I can see it presented in both ways which is part of the confusion. Most times the variant / option pricing only affects single purchase items and the subscription price is unaffected but in one instance it effects both. Are you saying this is a functional different just in Horizon? Or something configurable in the backend API?

https://swell-horizon-demo.vercel.app/products/hydrating-clay-mask (subscription unaffected by options)
https://swell-horizon-demo.vercel.app/products/moisturizer-spf-15 (works with both)

**Reply by ericingram** [reply]
cc @ana-estrela who designed and led the development of Horizon, maybe she can better answer those questions

Re: @fucha-co I'm not sure without digging into it (been a bit busy with another big product iteration), but I think we did implement subscriptions in Origin but perhaps it's just not in the demo. Ana could also help confirm that

**Reply by ana-estrela** [reply]
Hey @holden. the difference between the two products was a simple content population oversight on our end. You're now able to see the same functionality on both products.
